tests/nvmeof/test_ceph_nvmeof_neg_tests.py

```diff
@@ -919,14 +920,8 @@ def test_ceph_83581753(ceph_cluster, rbd, pool, config):
         qos_args_with_invalid_values.setdefault("args", {}).update(
             {"nsid": 10, "subsystem": subsystem["nqn"], "rw-ios-per-second": 10}
         )
-        try:
-            _ = nvmegwcli.namespace.set_qos(**qos_args_with_invalid_values)
-        except Exception as err:
-            if "Can't find namespace" not in str(err):
-                raise Exception(
-                    "Set QoS was failed as expected due to invalid namespace."
-                )
-            LOG.info("Set QoS was failed as expected due to invalid namespace....")
+        _ = nvmegwcli.namespace.set_qos(**qos_args_with_invalid_values)
```

Comment:
W.r.t previoud code line, use json format output which works with namespace list, use that validate the test. The idea is to avoid regex.